policies/voting-procedure.md

```diff
@@ -0,0 +1,72 @@
+The Public Voting Procedure of OTC
+==================================
+
+The following regulations complement the [OTC Voting Procedures] stated
+in the project bylaws. This policy affects only public votes.
+
+[OTC Voting Procedures]: https://www.openssl.org/policies/omc-bylaws.html#otc-voting
+
+Vote Proposal
+-------------
+
+The votes are proposed in pull requests and issues of the technical-policies
+repository on GitHub OpenSSL project.
+
+The vote regarding a policy change proposal is recorded directly in the
+pull request of the policy change proposal.
+
+Any other votes are recorded as separate issues in the repository.
+
+Casting the Votes
+-----------------
+
+The individual members of OTC cast their votes directly in the issue or
+pull request where the vote was proposed.
+
+When the votes are cast during an OTC meeting, the person responsible for
+taking the meeting minutes or the proposer of the vote copies the votes
+cast during the meeting into the issue.
+
+The Final Vote Records
+----------------------
+
+After the vote is closed the proposer of the vote records the final outcome
+and individual member votes in a separate file in the votes subdirectory of
+the repository. The commit with the vote record is pushed directly into
+the master branch of the repository. No pull request is needed.
+
+The file is formatted as follows:
+
+```
+topic: .
+Proposed by .
```

Comment:
Should this have a link to the PR or issue?

Reply:
A URL to the PR or issue holding the details of whatever is being voted on should be either in the topic itself as a requirement or as a separate named field.

Reply:
I'll add that